mohanadarafe/GuideMe
# fix: [#110]

Fixed a performance issue where the image would take a few seconds to load. 

### To test:
1. Checkout branch
`git checkout bug/#110
`
2. Click on a building
3. Slide up the MoreDetails component
4. The image should load in no time :)

Fixes #110
